tests/test_stac.py

```diff
@@ -155,3 +155,6 @@ def test_create_item_2022(self):
         self.assertEqual(item.id, "ca_m_4112001_sw_10_060_20220716")
         schemas = item.validate()
         self.assertTrue(schemas)
+
+        assert item.properties["proj:centroid"]["lat"] is float
```

Comment:
```suggestion
        assert isinstance(item.properties["proj:centroid"]["lat"], float)
```

or maybe `assert type(item.properties["proj:centroid"]["lat"] == float` if we specifically want python floats and not subclasses (I'm not sure if `numpy.float64` subclasses `float`)

Reply:
`numpy.float64` does subclass `float`. Initially I had a call to `isinstance` but that was working even for the numpy type whereas `is` strictly checks for exact types.

Reply:
Gotcha. I think you'll want `assert type(item.properties["proj:centroid"]["lat"] == float` then. You'll need to be careful with `is` checks. Typically it's just used for things that are known to be singletons like `None`, `True`, and `False`, or if you want to check that that two names are actually references to the same object.